### ISSUE TITLE
Ports 9009 open after wii connect

### ISSUE THREAD

**Issue by PhynixP** [barronwaffles/dwc_network_server_emulator#130]
Hello i have a little problem with the newest dwc server
i use a Raspberry for my Testproject.

First, my wii und my dophin emu with mariokartwii, connecting fine with the server an i can play.

Now my Problem.
when i start the sever and i go to the adminpage ( IP:9009/banhammer ) i can't see this this site.
when i start the server an connect with my wii or dolphin, i can connect to the Site

why is the port closed without a connection from any device ?

**Comment by kyle95wm** [comment]
For whatever reason that's always been the case and I'm not sure why that happens but I remember when I used to run the server without Apache (I changed the listening port from 9000 to 80) I think the pages loaded without having to make a successful login however there's a problem with running master on port 80. 

These script kiddies run all these kinds of attacks on web servers so first thing you'll notice is weird messages popping up in the server logs. After a while your master server will simply stop working and will require a restart. To fix this I had to multi thread master which made it stay up for a bit longer. I stopped using this method since I wanted to run a home page for the server without killing the server every time.
